test/e2e/auth/audit.go

```diff
@@ -23,83 +23,623 @@ import (
+						"watch",
+						200,
+						auditTestUser,
+						"pods",
+						namespace,
+						false,
+						false,
+					}, {
+						v1beta1.LevelRequest,
+						v1beta1.StageResponseComplete,
+						fmt.Sprintf("/api/v1/namespaces/%s/pods?timeoutSeconds=%d&watch=true", namespace, watchTestTimeout),
+						"watch",
+						200,
+						auditTestUser,
+						"pods",
+						namespace,
+						false,
+						false,
+					}, {
+						v1beta1.LevelRequestResponse,
+						v1beta1.StageResponseComplete,
+						fmt.Sprintf("/api/v1/namespaces/%s/pods/audit-pod", namespace),
+						"update",
+						200,
+						auditTestUser,
+						"pods",
+						namespace,
+						true,
+						true,
+					}, {
+						v1beta1.LevelRequestResponse,
+						v1beta1.StageResponseComplete,
+						fmt.Sprintf("/api/v1/namespaces/%s/pods/audit-pod", namespace),
+						"patch",
+						200,
+						auditTestUser,
+						"pods",
+						namespace,
+						true,
+						true,
+					}, {
+						v1beta1.LevelRequestResponse,
+						v1beta1.StageResponseComplete,
+						fmt.Sprintf("/api/v1/namespaces/%s/pods/audit-pod", namespace),
+						"delete",
+						200,
+						auditTestUser,
+						"pods",
+						namespace,
+						true,
+						true,
+					},
+				},
+			}, {
+				"Deployment",
+				func() {
+					podLabels := map[string]string{"name": "audit-deployment-pod"}
+					d := framework.NewDeployment("audit-deployment", int32(1), podLabels, "redis", imageutils.GetE2EImage(imageutils.Redis), extensions.RecreateDeploymentStrategyType)
+
+					_, err := f.ClientSet.Extensions().Deployments(f.Namespace.Name).Create(d)
+					framework.ExpectNoError(err, "failed to create audit-deployment")
+
+					_, err = f.ClientSet.Extensions().Deployments(f.Namespace.Name).Get(d.Name, metav1.GetOptions{})
+					framework.ExpectNoError(err, "failed to get audit-deployment")
+
+					deploymentChan, err := f.ClientSet.Extensions().Deployments(f.Namespace.Name).Watch(watchOptions)
+					for range deploymentChan.ResultChan() {
+					}
+					framework.ExpectNoError(err, "failed to create watch for deployments")
+
+					_, err = f.ClientSet.Extensions().Deployments(f.Namespace.Name).Update(d)
+					framework.ExpectNoError(err, "failed to update audit-deployment")
+
+					_, err = f.ClientSet.Extensions().Deployments(f.Namespace.Name).Patch(d.Name, types.JSONPatchType, patch)
+					framework.ExpectNoError(err, "failed to patch deployment")
+
+					_, err = f.ClientSet.Extensions().Deployments(f.Namespace.Name).List(metav1.ListOptions{})
+					framework.ExpectNoError(err, "failed to create list deployments")
+
+					err = f.ClientSet.Extensions().Deployments(f.Namespace.Name).Delete("audit-deployment", &metav1.DeleteOptions{})
+					framework.ExpectNoError(err, "failed to delete deployments")
+				},
+				[]auditEvent{
+					{
+						v1beta1.LevelRequestResponse,
+						v1beta1.StageResponseComplete,
+						fmt.Sprintf("/apis/extensions/v1beta1/namespaces/%s/deployments", namespace),
```

Comment:
Shouldn't it be in the `apps` api group?

Reply:
Yes, but in test I'm using extensions api (which I assume is older version for deployments). That's why we see /apis/extensions/v1beta1 in the output.